mix_tasks/scraper.ex

```diff
@@ -2,13 +2,18 @@ defmodule Location.Scraper do
   @base_url "https://en.wikipedia.org"
   @subdivision_base_url @base_url <> "/wiki/ISO_3166-2:"
   @translations_dest Application.app_dir(:location, "/priv/iso_3166-2.en-translations.json")
+  @version_file Application.app_dir(:location, "/priv/version")
```

Comment:
At compile time this will resolve to `/app/_build/prod/lib/location/priv/` which doesn't exist in the release target. At runtime, you'll get an existing directory, but no write permissions to it (same goes for `@translations_dest`).

Reply:
https://github.com/plausible/location/pull/25#discussion_r1671809251